charts/trino/values.yaml

```diff
@@ -948,6 +948,8 @@ serviceMonitor:
   # serviceMonitor.enabled -- Set to true to create resources for the
   # [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator).
   enabled: false
+  # serviceMonitor.apiVersion -- API version for serviceMonitor
```

Comment:
Remove the docs, they're not helpful - it's basically the same as the property name.